beam-postgres/Database/Beam/Postgres/#Connection.hs#

```diff
@@ -0,0 +1,394 @@
+{-# OPTIONS_GHC -fno-warn-orphans -fno-warn-partial-type-signatures #-}
```

Comment:
Perhaps we need a .gitignore for this kind of file.

Reply:
Yeah...